subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/fixture/JUnitMultiVersionIntegrationSpec.groovy

```diff
@@ -92,13 +92,13 @@ abstract class JUnitMultiVersionIntegrationSpec extends MultiVersionIntegrationS
         }
     }
 
-    protected getDependencyNotation() {
+    protected List<String> getDependencyNotation() {
```

Comment:
Is there some way we can refactor this so that we eliminate the need for these explicit dependencies too?

IOW, instead of calling `dependencyNotation` everywhere, we'd use test suites to switch between JUnit and JUnit Jupiter. Only in the case of the vintage engine we'll need to add extra dependencies?

Reply:
Perhaps, but I'd like to do that as part of a separate PR.